src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,16 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1024px;
```

Comment:
13. [LAYOUT] - Use sass functions for dynamic max-width calculation, your catalog container should work fine for any number of cards.